common/credentialFactory.go

```diff
@@ -21,6 +21,7 @@
 package common
 
 import (
+	"cloud.google.com/go/storage"
 	"context"
```

Comment:
Consider using an aliased import instead of the stock storage package name-- Perhaps `gcp` would be a good fit?

Reply:
Changing this to gcpUtils. Storage is indeed confusing in this code.